exercises/clock/clock_test.rb

```diff
@@ -145,4 +145,81 @@ def test_wrap_around_backwards
     clock = Clock.at(0, 30) - 60
     assert_equal '23:30', clock.to_s
   end
+
+  def test_clocks_with_same_time
+    skip
+    clock1 = Clock.at(15, 37)
+    clock2 = Clock.at(15, 37)
+    assert_equal clock1, clock2
+  end
+
+  def test_clocks_a_minute_apart
+    skip
+    clock1 = Clock.at(15, 36)
+    clock2 = Clock.at(15, 37)
+    refute_equal clock1, clock2
+  end
+
+  def test_clocks_an_hour_apart
+    skip
+    clock1 = Clock.at(14, 37)
+    clock2 = Clock.at(15, 37)
+    refute_equal clock1, clock2
+  end
+
+  def test_clocks_with_hour_overflow
+    skip
+    clock1 = Clock.at(10, 37)
+    clock2 = Clock.at(10, 37) + days(1) + hours(1)
+    refute_equal clock1, clock2
+  end
+
+  def test_clocks_with_hour_overflow_by_several_days
+    skip
+    clock1 = Clock.at(3, 11)
+    clock2 = Clock.at(0, 11) + hours(99)
+    assert_equal clock1, clock2
```

Comment:
The reason I added the tests in the first place was that in the Go track I was seeing bugs that the tests didn't catch; particularly clocks that were not equal.

I'd like to loosen up the restrictions on the example (which I think maybe you started working on) so that the tests catch this sort of thing if people take a different approach that we currently do in the example.